eng/ci/build-core-tools-host-artifacts-windows.yml

```diff
@@ -1,89 +1,42 @@
-trigger:
-  branches:
-    include:
-    - feature/oop-host
-  paths:
-    include:
-      - ./host/src/**
+pr: none
 
-pr:
+trigger:
   branches:
     include:
     - feature/oop-host
   paths:
     include:
-      - ./host/src/**
+      - /host/src/*
+      - /host/src/*/*
+      - /host/src/*/*/*
```

Comment:
does `/host/src/**` not work?

Reply:
no it wasn't working before :( I had it has `./host/src/**` before though so maybe that might be the reason why? I wouldn't think that the . in the beginning would change anything

Reply:
Changed to `/host/src/**`